src/styles/main.scss

```diff
@@ -1,3 +1,182 @@
-body {
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
```

Comment:
It is forbidden to set sizes and margins for blocks, but for block elements are allowed. The block should stretch to accommodate its children